src/questionnaireitem_media_video.js

```diff
@@ -40,11 +42,28 @@ QuestionnaireItemMediaVideo.prototype._createAnswerNode = function() {
 };
 
 QuestionnaireItemMediaVideo.prototype.releaseUI = function() {
-    this.node = null;
```

Comment:
Please leave it on top. It will be replaced by a super()-call when transformation to ES6 class structure is done.

Reply:
Restored original implementation.